machinery/importGraph.test.js

```
@@ -0,0 +1,532 @@
+const { describe, it, afterEach } = require('node:test')
+const assert = require('node:assert')
+const { writeFileSync, mkdirSync, rmSync } = require('node:fs')
+const { join } = require('node:path')
+const { getImportGraph, _resetCache } = require('./importGraph')
+
+const fixtureRoot = join(__dirname, '..', '.test-fixtures', 'import-graph')
+
+afterEach(() => {
+  _resetCache()
+  try { rmSync(fixtureRoot, { recursive: true, force: true }) } catch {}
```

ESLint

[error] Empty block statement.

(no-empty)